tests/integration/test_project.py

```diff
@@ -97,6 +97,41 @@ def test_project_export_v2(configured_project_with_label):
     assert task_from.status == "COMPLETE"
 
 
+@pytest.mark.parametrize("datarows", [3], indirect=True)
+def test_project_export_v2_datarow_list(
+        configured_batch_project_with_multiple_datarows):
+    batch_project, _, datarows = configured_batch_project_with_multiple_datarows
+    data_row_ids = [dr.uid for dr in datarows]
+    batch_project._wait_until_data_rows_are_processed(
```

Comment:
aha... wonder whether this method has the same effect as `wait_for_data_row_processing`, cc @kkim-labelbox